explainer.md

```diff
@@ -235,40 +171,194 @@ input.activeDescendantElement = optionList.firstChild;
+  <custom-tab selected>Tab 1</custom-tab>
+  <custom-tab>Tab 2</custom-tab>
+  <custom-tab>Tab 3</custom-tab>
+</div>
+```
+
+If the author-provided semantics conflict with the Custom Element semantics,
+the author-provided semantics take precedence.
 
-* `accessibleclick`
-* `accessiblecontextmenu`
-* `accessiblefocus`
-* `accessiblesetvalue`
-* `accessibleincrement`
-* `accessibledecrement`
-* `accessibleselect`
-* `accessiblescroll`
-* `accessibledismiss`
+### User action events from Assistive Technology
+
+To preserve the privacy of assistive technology users,
+events from assistive technology will typically cause a synthesised DOM event to be triggered:
+
+| **AT event**     | **Targets**                                                                        | **DOM event**                                                                   |
+|------------------|------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| `click`          | *all elements*                                                                     | `click`                                                                         |
+| `focus`          | *all elements*                                                                     | `focus`                                                                         |
+| `select`         | Elements whose mapped role is `cell` or `option`                                   | `click`                                                                         |
+| `scrollIntoView` | (n/a)                                                                              | No event                                                                        |
+| `dismiss`        | *all elements*                                                                     | Keypress sequence for `Escape` key                                              |
+| `contextMenu`    | *all elements*                                                                     | `contextmenu`                                                                   |
+| `scrollByPage`   | *all elements*                                                                     | Keypress sequence for `PageUp` or `PageDown` key, depending on scroll direction |
+| `increment`      | Elements whose mapped role is `progressbar`, `scrollbar`, `slider` or `spinbutton` | Keypress sequence for `Up` key                                                  |
+| `decrement`      | Elements whose mapped role is `progressbar`, `scrollbar`, `slider` or `spinbutton` | Keypress sequence for `Down` key                                                |
+| `setValue`       | Elements whose mapped role is `combobox`,`scrollbar`,`slider` or `textbox`         | TBD                                                                             |
```

Comment:
`setValue` — a bunch of keyboard events? maybe? :D

Reply:
Would be weird for anything other than a textbox.